source/code/scxsystemlib/disk/diskdepend.cpp

```diff
@@ -547,7 +568,8 @@ namespace SCXSystemLib
             if (line.find(L"loop=") != std::wstring::npos || line.find(L"/dev/loop") != std::wstring::npos)
             {
                 // for Linux, ignore files mounted as devices using the loopback driver
-                continue;
+                if (!isTestEnv)
```

Comment:
Initialize the isTestEnv variable inside this if loop. It is not required to have it initialized during function invocation.

Reply:
Adding inside this if loop will make call to FilExists in loop.